src/komponenter/PakrevdInput/PakrevdInput.tsx

```diff
@@ -1,42 +1,61 @@
 import React, { useState, ChangeEvent } from 'react';
 import { Input } from 'nav-frontend-skjema';
 import { SkjemaelementFeil } from 'nav-frontend-skjema/lib/skjemaelement-feilmelding';
-import { bool } from 'prop-types';
 
 interface Props {
     className?: string;
     label: React.ReactNode;
     verdi: string;
     feilmelding?: string;
     ekstraValidering?: boolean;
+    inputType?: string;
+    validatePhoneNr?: boolean;
     // * onChange bør oppdatere verdi feltet på props
     onChange: (event: ChangeEvent<HTMLInputElement>) => void;
 }
 
 const PakrevdInput: React.FunctionComponent<Props> = props => {
```

Comment:
I stedet for å lage ny prop i `PakrevdInput`, kan du bruke propen `ekstraValidering` som er en funksjon som utfører mer validering enn truthy/falsy. Kunne laget ny komponent `PakrevdTelefonInput`, for å unngå at `PakrevdInput` skal håndtere potensielt veldig mange inputs.